cli/cook/http.py

```diff
@@ -45,6 +45,12 @@ def configure(config):
             raise Exception(f'Encountered unsupported authentication type "{auth_type}".')
 
 
+def __head(url, params=None, **kwargs):
+    """Sends a HEAD with params to the given url"""
+    logging.info(f'HEAD {url} with {params}')
+    return session.head(url, params=params, timeout=timeouts, **kwargs)
+
+
```

Comment:
Where is this used?

Reply:
The plugin we're writing uses a HEAD request to check if the data exists, and falls back to the default command if it doesn't. It's not actually used anywhere inside Cook itself.